apps/docs/src/guide/cli/config-file.md

```diff
@@ -90,6 +90,20 @@ Port to use when starting a local `fuel-core` node.
 
 <<< @../../../demo-fuels/fuels.config.full.ts#fuelCorePort{ts:line-numbers}
 
+## `forcBuildFlags`
+
+> - _Used by [`fuels build`](./commands.md#fuels-build) and [`fuels deploy`](./commands.md#fuels-deploy)_.
+
+Sway programs are compiled in `debug` mode by default.
+
+Here you can customize all build `flags`, e.g. to build programs in `release` mode.
```

Comment:
```suggestion
Here you can customize all build flags, e.g. to build programs in `release` mode.
```
I think "flags" shouldn't be in backticks here because it's used in its standard meaning.